### PR TITLE
Bug/touch icon 404

### PR DESCRIPTION
**Pull request by robertparkinson**
### Change description
Move touch icon into root directory to stop 404s from devices that assume it is there.
Update security policy to allow inline js 

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,10 +18,10 @@
     "test:coverage-unit": "TS_NODE_TRANSPILE_ONLY=true NODE_ENV=mocha LOG_LEVEL=OFF nyc mocha --opts mocha.opts $(find src/test \\( -name '*.ts' \\) ! -path '*a11y*' ! -path '*/routes/*')",
     "test:codacy-upload": "cat ./coverage-report/lcov.info | codacy-coverage",
     "test:codecov-upload": "codecov -f ./coverage-report/lcov.info",
-    "test:smoke": "./bin/run-smoke-tests.sh",
-    "test:integration": "TS_NODE_TRANSPILE_ONLY=true ts-node --require tsconfig-paths/register ./node_modules/.bin/codeceptjs run --verbose --config default.conf.js --reporter mocha-multi",
-    "test:integration-debug": "TS_NODE_TRANSPILE_ONLY=true ts-node --require tsconfig-paths/register ./node_modules/.bin/codeceptjs run --verbose --config default.conf.js",
-    "test:cross-browser": "TS_NODE_TRANSPILE_ONLY=true ts-node --require tsconfig-paths/register ./node_modules/.bin/codeceptjs run --verbose --config saucelabs.conf.js --reporter mocha-multi",
+    "test:smoke": "echo disabled due to testing breaking change to journey",
```
Comment (rlewan) [inline package.json]: I think this may be a merging problem, we probably want to execute the smoke tests via this task?

File: src/main/modules/helmet/modules/contentSecurityPolicy.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export class ContentSecurityPolicy {
   constructor (public developmentMode: boolean) {}
 
   enableFor (app: express.Express) {
-    const scriptSrc = [self, '*.google-analytics.com']
+    const scriptSrc = ['\'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=\'', '\'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=\'', self, '*.google-analytics.com']
```
Comment (rlewan) [inline src/main/modules/helmet/modules/contentSecurityPolicy.ts]: This change does not seem to be related to the missing icon problem, what is it for?

Comment (robertparkinson) [reply src/main/modules/helmet/modules/contentSecurityPolicy.ts]: we noticed it when we were looking at errors returned to the console and the network tab so seemed as good a place to fix them. Am happy to pull them out into a separate PR

File: package.json (hunk 1 above)
Comment (geek-soft) [reply package.json]: Please revert

File: package.json [hunk 3]
```diff
@@ -18,10 +18,10 @@
...
+    "test:integration": "TS_NODE_FAST=true ts-node --require tsconfig-paths/register ./node_modules/.bin/codeceptjs run --verbose --config default.conf.js --reporter mocha-multi",
```
Comment (geek-soft) [inline package.json]: Please revert. After ts-node upgrade `TS_NODE_FAST` got replaced with `TS_NODE_TRANSPILE_ONLY`

File: src/main/modules/helmet/modules/contentSecurityPolicy.ts (hunk 2 above)
Comment (geek-soft) [reply src/main/modules/helmet/modules/contentSecurityPolicy.ts]: I would love to have it in separate PR.

File: package.json (hunk 1 above)
Comment (robertparkinson) [reply package.json]: have closed this PR as I used the wrong branch to base my PR off. this branch is what we use for demo.
